
~~~PATCH 5879fa7b665389d839ae149d551d8860:c151bafe3d209c0af44a534a64d9c9b8 lib/Foswiki/Sandbox.pm (Foswiki-1.1.8)
Date:   Sat Mar 2 03:58:37 2013 +0000

    Item12414: tempfile is not an object method.
    
    Next version of File::Temp will die if called as File::Temp->tempfile()
    
    git-svn-id: http://svn.foswiki.org/branches/Release01x01@16579 0b4bb1d4-4e5a-0410-9cc4-b2b747904278

diff --git lib/Foswiki/Sandbox.pm lib/Foswiki/Sandbox.pm
index fc0312b..9a465d5 100644
--- lib/Foswiki/Sandbox.pm
+++ lib/Foswiki/Sandbox.pm
@@ -503,7 +503,7 @@ sub sysCommand {
 
     # Note:  Use of the file handle $fh returned here would be safer than
     # using the file name. But it is less portable, so filename wil have to do.
-    my ( $fh, $stderrCache ) = File::Temp->tempfile(
+    my ( $fh, $stderrCache ) = File::Temp::tempfile(
         "STDERR.$$.XXXXXXXXXX",
         DIR    => "$Foswiki::cfg{WorkingDir}/tmp",
         UNLINK => 0
